test/integration/targets/playbook/runme.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -eux
+
+# run type tests
+ansible-playbook -i ../../inventory types.yml -v "$@"
```

Comment:
Why not use a role instead of a script based test?

Reply:
mostly because i thought that was the 'new norm', but also because these are easier to expand and less subject to other vars/effects created in same invocation.

Reply:
We're only using them as needed. There's nothing to be "separate" from. Each role is executed in its own playbook, so using a script doesn't provide any more isolation than a role.

Reply:
but all tests for playbook would have to be in same role, that is what i meant.

Reply:
If you're planning on adding additional tests in separate playbooks for this target, then using a script makes sense. I was mainly asking because right now there's just the one ansible-playbook invocation, so no benefit to using a script.

Reply:
right now this just checks booleans (and a specific field at that), but my thought was that we should test all the rest also.